security/sec-iride/src/test/java/org/geoserver/security/iride/identity/IrideIdentityTokenizerTest.java

```diff
@@ -80,7 +79,7 @@ public void testTokenizationThrowNullPointerExceptionForNullValue() throws Iride
      */
     @Test(expected = IrideIdentityMissingTokenException.class)
     public void testTokenizationFailForEmptyOrBlankValue() throws IrideIdentityMissingTokenException {
-        final String value = StringUtils.repeat(" ", RandomUtils.nextInt(10));
+        final String value = randomBlankOrEmptyString();
```

Comment:
@mbarto I have "splitted" those unit test relying on random length strings, to have simpler, more focused, tests